db/migrate/20180606155924_move_ansible_container_secrets_into_database.rb

```diff
@@ -54,7 +54,7 @@ def update_authentications
     database_auth_find_args = db_args.merge(
       :name     => "Ansible Database Authentication",
       :authtype => "ansible_database_password",
-      :userid   => ApplicationRecord.configurations[Rails.env]["username"],
+      :userid   => ApplicationRecord.configurations.configs_for(:env_name => Rails.env).first.configuration_hash["username"],
```

Comment:
wow the new way is gross - wonder if we should move it into a spec helper method.

Reply:
I'll check to see if there's a less gross way to do this.  Although, I believe we had changes like this for appliance console and other areas already.

Reply:
a little better but still the way rails wants it:

```ruby
ApplicationRecord.configurations.configs_for(:env_name => Rails.env, :name => "primary").configuration_hash["username"]
```